File: C# Basics/NestedConditions/Cinema.cs
using System;

namespace Ex03_NestedConditions
{
    class Program
    {
        static void Main(string[] args)
        {
            string movieType = Console.ReadLine();
            int rows = int.Parse(Console.ReadLine());
            int columns = int.Parse(Console.ReadLine());

            double maxIncome = 0;

            switch (movieType)
            {
                case "Premiere":
                    maxIncome = rows * columns * 12;
                    break;
                case "Normal":
                    maxIncome = rows * columns * 7.5;
                    break;
                case "Discount":
                    maxIncome = rows * columns * 5;
                    break;
            }

            Console.WriteLine($"{maxIncome:f2}");

        }
    }
}

File: Java Basics/ConditionsAdvancedEx/Cinema.java
package AdvancedConditionsEx;

import java.util.Scanner;

public class Cinema {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String type = scanner.nextLine();
        int rows = Integer.parseInt(scanner.nextLine());
        int columns = Integer.parseInt(scanner.nextLine());

        double income = rows * columns;

        switch (type) {
            case "Premiere":
                income *= 12;
                break;
            case "Normal":
                income *= 7.5;
                break;
            case "Discount":
                income *= 5;
                break;
        }

        System.out.printf("%.2f leva", income);
    }
}
